.taskcluster.yml

```diff
@@ -0,0 +1,35 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  - taskId: {$eval: as_slugid("tests")}
```

Comment:
Out of curiosity, where is `as_slugid` defined? I wasn't able to find it in https://docs.taskcluster.net/docs/reference/integrations/taskcluster-github/docs/taskcluster-yml-v0.

Reply:
It's in https://docs.taskcluster.net/docs/reference/integrations/taskcluster-github/docs/taskcluster-yml-v1 :)

Reply:
Ahh. Btw, https://tools.taskcluster.net/quickstart defaults to version 0. Should this be updated to version 1?